### PR TITLE
Use dropdown search for tablet size

### LINKED ISSUE #4526: search results: use dropdown for tablet size
>The issue with changing it from 1024 to 768 is that we have the mobile navbar there. But i >think it makes much more sense to use only the popup than the full screen. Because you >have option to click out of the results (which you dont have, or is hard on mobile) 

>Sooo, i am not against using dropdown/popup for tablet view and then for the mobile the full >page results. 

_Originally posted by @exezbcz in https://github.com/kodadot/nft-gallery/issues/4519#issuecomment-1356822553_

### PR DESCRIPTION
**Pull request by iMac7**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #4526 , #8824 
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=14dLrAvjDw7cFLaxM7VWR6YtHd7bGZY2QpzXDhAJJ4yv6oGw&amount=0)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screenshot_20240226_200529

This will probably need some designs? I used some opinionated values. 
Added a min width because the dropdown gets too small sometimes when approaching tablet width.
Not sure if the logo should be present in tablet.
The mobile dropdown for search is not currently visible on canary  :eyes: , so this fix doesn't cover mobile, lmk if it should.

### CODE REVIEW

**Comment by exezbcz** [comment]
quite old issue, will check soon

**Comment by prury** [comment]
well it does seem better than canary, but maybe some design changes? let's wait for exez

1000px width: 
image

**Comment by exezbcz** [comment]
@iMac7 i dont think there should be "cancel" button shown - as on the mobile, it should stay in desktop style till some breakpoint 

- will comment in few hours with more detailed comment

**Comment by exezbcz** [comment]
> @iMac7 i dont think there should be "cancel" button shown - as on the mobile, it should stay in desktop style till some breakpoint
> 
> * will comment in few hours with more detailed comment

it should be applied  from 1024 to 768px width only. That can still be a larger tablet. After that, it does not change and the navbar should have the mobile version 
image
- if you zoom in enough, you can actually see something is popping up under the search: 
image

on this breakpoint, it should show the popup, but maintain the mobile navbar
- there i would like to see this 
image


does that make sense? 



yours does not stretch the whole width 
image
- there is still the burger menu button

**Comment by prury** [comment]
still being applied to < 768 width

image

**Comment by exezbcz** [comment]
> maybe another issue or to fix on this PR?

would you like to fix?

**Comment by exezbcz** [comment]
should not be dropdown stretched but fullwidth screen

**Comment by exezbcz** [comment]
@iMac7 
image
image
not really

**Review by preschian — APPROVED** [review]
code lgtm

**Comment by prury** [comment]
search test broke, will try to fix it later on: 

image

File: components/search/SearchSuggestion.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    class="search-suggestion-container"
+    class="search-suggestion-container max-sm:pb-4"
     @click="resetSelectedIndex"
```
Comment (prury) [inline components/search/SearchSuggestion.vue:4]:
```suggestion
    data-testid="search-suggestion-container"
    @click="resetSelectedIndex"
```

**Comment by prury** [comment]
@iMac7 can you do me a favor and change `class` to `data-testid` in this file? https://github.com/kodadot/nft-gallery/blob/6684c9351c660a6f33506125b662724a2e9fe7c3/tests/e2e/search.spec.ts#L17

**Comment by yangwao** [comment]
cc @prury ?

image

**Comment by prury** [comment]
> cc @prury ?
> 
> image
> 

 i asked for changes in a specific file in a comment above and they were not done yet

**Comment by prury** [comment]
> @iMac7 can you do me a favor and change `class` to `data-testid` in this file? https://github.com/kodadot/nft-gallery/blob/6684c9351c660a6f33506125b662724a2e9fe7c3/tests/e2e/search.spec.ts#L17 

@ Hehe, i asked for changes in this file above as well, on the search.spec.ts, just to fix the test, but it's not showing for me on the changes files on PR,

**Comment by yangwao** [comment]
Thanks @iMac7 !
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 9.86 USD/DOT ~ 5.071 $DOT
🧗 [14dLrAvjDw7cFLaxM7VWR6YtHd7bGZY2QpzXDhAJJ4yv6oGw](https://polkadot.subscan.io/account/14dLrAvjDw7cFLaxM7VWR6YtHd7bGZY2QpzXDhAJJ4yv6oGw)
🔗 [0x344af6f2078274892b9f401676160c09a4120860a8f10f80df264bf40347d94a](https://polkadot.subscan.io/extrinsic/0x344af6f2078274892b9f401676160c09a4120860a8f10f80df264bf40347d94a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**